nemo_text_processing/text_normalization/en/graph_utils.py

```diff
@@ -35,9 +35,9 @@
 NEMO_ALPHA = pynini.union(NEMO_LOWER, NEMO_UPPER).optimize()
 NEMO_ALNUM = pynini.union(NEMO_DIGIT, NEMO_ALPHA).optimize()
 NEMO_HEX = pynini.union(*string.hexdigits).optimize()
-NEMO_NON_BREAKING_SPACE = u"\u00A0"
```

Comment:
Wait how were we functioning with the wrong unicode escape?

Reply:
I just double-checked. Line 38 on my end looks fine:
`NEMO_NON_BREAKING_SPACE = "\u00A0"`